tests/telemetry/test_telemetry.py

```diff
@@ -0,0 +1,568 @@
+import pytest
+
+from askui.telemetry import InMemoryProcessor, Telemetry, TelemetrySettings
+
+
+def test_telemetry_disabled():
```

Comment:
If you adding now tests. Do we want to setup a github actions for testing?

Reply:
Wanted to add this later but I added it now as the earlier the better